src/components/screens/map-screen/map-screen.tsx

```diff
@@ -725,6 +735,31 @@ class MapScreen extends React.Component<Props, State> {
     });
   }
 
+  /**
+   * Matches pollutant control map center with the main map center
+   */
+  private updatePollutantControl = () => {
+    const { mapViewport } = this.state;
+
+    if (!mapViewport.center) {
+      return;
+    }
+
+    this.setState({
+      pollutantControlMapCenter: mapViewport.center
+    });
+  }
+
+  /**
+   * Changes heatmap layer visibility
+   */
+  private changeHeatmapLayerVisibility = () => {
+    const { heatmapLayerVisible } = this.state;
+    this.setState({
```

Comment:
```suggestion
    const { heatmapLayerVisible } = this.state;
    
    this.setState({
```
Empty line before setState({})